### PR TITLE
Create an example on generating and scraping kube_pod_labels

### PR DESCRIPTION
**Pull request by petewall**

### CODE REVIEW

File: examples/kube-pod-labels/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Kube Pod Labels
+
+This example shows how to include Pod labels as metrics.
+
+Labels and annotations on Kubernetes objects are not set as metric labels on metrics like [kube_pod_info](https://github.com/kubernetes/kube-state-metrics/blob/main/docs/pod-metrics.md). This is
+because it would greatly increase metric cardinality, which can get costly. [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics)
+can optionally create additional metrics, like `kube_pod_labels` that create label metrics that can then be joined to
```
Comment (BeverlyJaneJ) [inline examples/kube-pod-labels/README.md:7]:
```suggestion
can optionally create additional metrics. For example, `kube_pod_labels` create label metrics that can then be joined to
```

File: examples/kube-pod-labels/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+```
+And this section tells the Grafana Agent to include that metric in the set that gets sent to Prometheus for storage:
+```yaml
+metrics:
+  kube-state-metrics:
+    metricsTuning:
+      includeMetrics: [kube_pod_labels]
+```
+
+Additional fields can be added to `metricLabelsAllowlist` to get label metrics for other objects, like Namespaces, Nodes, etc...
```
Comment (BeverlyJaneJ) [inline examples/kube-pod-labels/README.md:24]:
```suggestion
Additional fields can be added to `metricLabelsAllowlist` to get label metrics for other objects, such as Namespaces, Nodes, etc.
```

File: examples/kube-pod-labels/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Also, kube-state-metrics can also generate metrics for annotations with `metricAnnotationsAllowList`.
```
Comment (BeverlyJaneJ) [inline examples/kube-pod-labels/README.md:25]:
```suggestion
Also, kube-state-metrics can generate metrics for annotations with `metricAnnotationsAllowList`.
```

**Review by BeverlyJaneJ — APPROVED** [review]

**Comment by petewall** [comment]
For anyone looking for the latest link, its here:
https://github.com/grafana/k8s-monitoring-helm/tree/main/charts/k8s-monitoring/docs/examples/pod-labels-and-annotations

**Pull request merged**